resources/sass/app.scss

```diff
@@ -22,49 +22,49 @@ body {
 */
 main {
     max-width: 1090px;
-    padding: 0 $dimension-layout-xsmall;
+    padding: 0 1rem;
```

Comment:
Layout and spacing values were temporarily replaced with their explicit value until [T350189](https://phabricator.wikimedia.org/T350189) is done.